hadoop-hdfs-project/hadoop-hdfs/src/main/resources/hdfs-default.xml

```diff
@@ -2888,6 +2888,16 @@
   </description>
 </property>
 
+<property>
+  <name>dfs.pathname.forbidden.characters</name>
+  <value></value>
```

Comment:
value is empty?

Reply:
I thought some users might persist in using some special characters to mark path, so I leaf it empty. Maybe it's not a good idea, and I will update it in the next commit.

Reply:
Thank you for the review !